client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -225,14 +231,14 @@ export class SeoForm extends React.Component {
 		const { dirtyFields } = this.state;
 		const { trackFormSubmitted, trackTitleFormatsUpdated, trackFrontPageMetaUpdated } = this.props;
 
-		trackFormSubmitted();
+		trackFormSubmitted( { path: this.props.path } );
 
 		if ( dirtyFields.has( 'seoTitleFormats' ) ) {
-			trackTitleFormatsUpdated();
+			trackTitleFormatsUpdated( { path: this.props.path } );
 		}
 
 		if ( dirtyFields.has( 'frontPageMetaDescription' ) ) {
-			trackFrontPageMetaUpdated();
+			trackFrontPageMetaUpdated( { path: this.props.path } );
```

Comment:
We could deconstruct `path` since it will simplify this code:

```javascript
const { path, trackFormSubmitted, trackTitleFormatsUpdated, trackFrontPageMetaUpdated } = this.props;

trackFormSubmitted( { path } );

if ( dirtyFields.has( 'seoTitleFormats' ) ) {
  trackTitleFormatsUpdated( { path } );
}

if ( dirtyFields.has( 'frontPageMetaDescription' ) ) {
  trackFrontPageMetaUpdated( { path } );
}
```